angrmanagement/ui/main_window.py

```diff
@@ -222,6 +222,7 @@ def _init_menus(self):
         self._help_menu = HelpMenu(self)
         self._plugin_menu = PluginMenu(self)
 
+        #self.menuBar().setNativeMenuBar(False)
```

Comment:
Commit this line as a separate PR (check if the current OS is macOS, and if so, call `setNativeMenuBar(False)`).

Reply:
Got it. This was actually accidentally left here, but I will still make a change for it.